logtail/logtail.go

```diff
@@ -577,6 +577,7 @@ var (
 	openBracketV = []byte("[v")
 	v1           = []byte("[v1] ")
 	v2           = []byte("[v2] ")
+	vJSON        = []byte("[v\x00JSON:")
```

Comment:
Why no trailing `]`? E.g. `[v\x00JSON] `? I'm sure there's a reason, I just can't intuit what it is. Document it?

Reply:
JSON is at-least self-terminating, so we don't absolutely need some out-of-band terminator. A trailing ']' is aesthetically nicer, but slightly more work later to strip it later.